docs/Manifest.toml

```diff
@@ -1823,6 +1946,38 @@ version = "3.7.2+0"
 uuid = "ca575930-c2e3-43a9-ace4-1e988b2c1908"
 version = "1.2.0"
 
+[[deps.NonlinearSolve]]
```

Comment:
I'm surprised that so many new deps are added from just updating the versions. Do you know which package is bringing these in?

Reply:
No... I was surprised too... 
No deps change in Project.toml. 
Is it possible that there are change in the Manifests due to what I did on other branches? (even if I updated them from this new branch). 

Does it change for you if you update locally? Maybe we shouldn't merge yet. This is weird.

Reply:
@Sbozzolo I have ClimaDiagnostics in the Diagnostic branch. Could that explain this?
(But I thought the Manifest would refresh as I checked out from main, just up ClimaLand from 0.12.3 to 0.12.4 and up the Manifest in docs and .buildkite)

Reply:
It might be mostly from ( ? )

```
[[deps.DiffEqCallbacks]]
deps = ["DataStructures", "DiffEqBase", "ForwardDiff", "Functors", "LinearAlgebra", "Markdown", "NLsolve", "Parameters", "RecipesBase", "RecursiveArrayTools", "SciMLBase", "StaticArraysCore"]
git-tree-sha1 = "ee954c8b9d348b7a8a6aec5f28288bf5adecd4ee"
deps = ["DataStructures", "DiffEqBase", "ForwardDiff", "Functors", "LinearAlgebra", "Markdown", "NonlinearSolve", "Parameters", "RecipesBase", "RecursiveArrayTools", "SciMLBase", "StaticArraysCore"]
git-tree-sha1 = "c959cfd2657d16beada157a74d52269e8556500e"
uuid = "459566f4-90b8-5000-8ac3-15dfb0a30def"
version = "2.37.0"
version = "3.6.2"
```

Reply:
not sure why we were on 2.37.0 - that was February 14 release. looks like many deps got updated.

Reply:
Let me try locally. It could be brought in by the DiffEqCallbacks update

Reply:
It happens when I update the environments locally too

Reply:
Ok I'll merge it then! Thanks

Reply:
`DiffEqCallbacks` now depends on `NonlinearSolve`, which brings a bunch of extra dependencies